flow/declarations/CardanoCrypto.js

```diff
@@ -139,23 +139,23 @@ declare module 'rust-cardano-crypto' {
 
 declare type CryptoWallet = {
   root_key: string,
-  cached_root_key: string,
+  root_cached_key: string, // cached_root_key -> root_cached_key
```

Comment:
i think it is kind of special case, may be a probable flow bug.
defined as `cached_root_key` but used as `root_cached_key`

made a special commit with removed comment,  may be it will help us dig later if needed.